### ISSUE TITLE
Opaque terminal and winblend

### ISSUE THREAD

**Issue by talbergs** [neovim/neovim#10685]
#### Terminal is transparent except for floating windows.
I'm not sure if this is a bug. In the snippet below, I create floating window, my terminal app is opaque, so I can see desktop background, that's not the case for floating window though. 

- NVIM v0.4.0-1534-g24fb7eefa | Build type: Debug
- Operating system/version: 5.1.2-arch1-1-ARCH | sway (wayland wm)
- Terminal name/version: termite v15

### Steps to reproduce using `nvim -u NORC`

```
let l = []| let w = winwidth(0)| let h = winheight(0)| let s = ""
while w > 0| let w -= 1| let s .= "."| endwhile
while h > 0| let h -= 1| let l += [s]| endwhile
call append("$", l)

set showtabline=2
set termguicolors

hi TabLine gui=NONE cterm=NONE
hi TabLineFill gui=NONE cterm=NONE
hi TabLineSel gui=NONE cterm=NONE
hi Normal guibg=NONE

let buf = nvim_create_buf(v:false, v:true)
let win = nvim_open_win(buf, v:true, {
            \'row':0,
            \'col':0,
            \'width':20,
            \'height':10,
            \'relative':'editor',
            \'focusable':v:true})

call nvim_win_set_option(win, "winblend", 100)
call nvim_buf_set_lines(buf, 0, 1, v:false, ["   TEXT", ""])
```

### Actual behaviour
Desktop background not visible only through floating window. 

### Expected behaviour
To be visible. (for winblend value of 100)

**Comment by bfredl** [comment]
It is because with `blend>0` we always blend the color. We could add an edge case for `blend=100` to preserve the back layer background exactly (including `bg=NONE`)

**Comment by justinmk** [comment]
@bfredl is this a blocker ? Seems like it could wait until 0.5

**Comment by bfredl** [comment]
Yeah it is just "nice to have". Most people do not use winblend=100

**Comment by Maswor** [comment]
Confirmed that the bug is still around for Alacritty on MacOS. It would be nice to have :)

**Comment by Chaitanyabsprip** [comment]
still an issue, should be prominent among transparent theme users along with noice/drop/notify, etc
